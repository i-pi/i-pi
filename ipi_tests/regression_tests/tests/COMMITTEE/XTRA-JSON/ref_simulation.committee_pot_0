 #EXTRAS(committee_pot)# Step:           0  Bead:      0  
    -2.04667050          -2.04134110
 #EXTRAS(committee_pot)# Step:           2  Bead:      0  
    -2.38304606          -2.38175991
 #EXTRAS(committee_pot)# Step:           4  Bead:      0  
    -2.91944545          -2.92399517
 #EXTRAS(committee_pot)# Step:           6  Bead:      0  
    -2.91834270          -2.92055288
 #EXTRAS(committee_pot)# Step:           8  Bead:      0  
    -2.53592815          -2.53052093
 #EXTRAS(committee_pot)# Step:          10  Bead:      0  
    -2.51777659          -2.51185574
