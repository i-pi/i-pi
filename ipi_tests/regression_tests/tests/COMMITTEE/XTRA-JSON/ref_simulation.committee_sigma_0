 #EXTRAS(committee_uncertainty)# Step:           0  Bead:      0  
0.003768454879655536
 #EXTRAS(committee_uncertainty)# Step:           2  Bead:      0  
0.0009094453866229897
 #EXTRAS(committee_uncertainty)# Step:           4  Bead:      0  
0.003217137864500044
 #EXTRAS(committee_uncertainty)# Step:           6  Bead:      0  
0.001562833265642613
 #EXTRAS(committee_uncertainty)# Step:           8  Bead:      0  
0.0038234819293677916
 #EXTRAS(committee_uncertainty)# Step:          10  Bead:      0  
0.004186673185388275
