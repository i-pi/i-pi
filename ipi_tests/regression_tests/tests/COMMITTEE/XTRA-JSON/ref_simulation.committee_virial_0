 #EXTRAS(committee_virial)# Step:           0  Bead:      0  
    -0.81755615          -0.67048751          -0.08631796          -0.68559468          -0.87989917          -0.23878892          -0.07289286           4.77220104          -1.39727383
    -0.63184142          -0.67048751          -0.08631796          -0.69146025          -0.69504091          -0.23878892          -0.06919554           4.76776664          -1.21531437
 #EXTRAS(committee_virial)# Step:           2  Bead:      0  
    -1.00250355          -0.40680685           0.21926609          -0.41525750          -1.59842499          -0.41869656           0.24637225           4.58962928          -1.84231923
    -0.82653631          -0.40680685           0.21926609          -0.41886929          -1.43879338          -0.41869656           0.25562172           4.58270562          -1.68210257
 #EXTRAS(committee_virial)# Step:           4  Bead:      0  
    -1.78087570           0.08874021           0.29705564           0.09690366          -2.40220776          -0.50456540           0.32243064           4.50986101          -2.22891158
    -1.63729716           0.08874021           0.29705564           0.09943184          -2.26972518          -0.50456540           0.33090440           4.50214358          -2.09116579
 #EXTRAS(committee_virial)# Step:           6  Bead:      0  
    -1.27168116           0.58411521           0.17567211           0.61126848          -1.78646241          -0.43474307           0.19225070           4.59546287          -1.15132577
    -1.11898260           0.58411521           0.17567211           0.62094114          -1.64425541          -0.43474307           0.19732091           4.58904099          -0.99535290
 #EXTRAS(committee_virial)# Step:           8  Bead:      0  
    -0.00482487           0.80543575           0.03198130           0.84579277          -0.34561059          -0.23309011           0.03851077           4.81711050           0.43713535
     0.18572398           0.80543575           0.03198130           0.86066186          -0.16894066          -0.23309011           0.03963847           4.81366117           0.63276429
 #EXTRAS(committee_virial)# Step:          10  Bead:      0  
    -0.09510212           0.54524386           0.00411663           0.57910363           0.03166231           0.00232131           0.01428536           5.06896150           0.44412525
     0.09321336           0.54524386           0.00411663           0.59146193           0.21550328           0.00232131           0.01734718           5.06918048           0.64240982
