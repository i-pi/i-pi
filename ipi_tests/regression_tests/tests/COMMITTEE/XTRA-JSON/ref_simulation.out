# column   1     --> step : The current simulation time step.
# column   2     --> time{picosecond} : The elapsed simulation time.
# column   3     --> conserved : The value of the conserved energy quantity per bead.
# column   4     --> temperature{kelvin} : The current temperature, as obtained from the MD kinetic energy.
# column   5     --> potential : The physical system potential energy.
# column   6     --> pressure_md{bar} : The pressure of the (extended) classical system.
# column   7     --> volume : The volume of the cell box.
    0.00000000e+00     0.00000000e+00    -1.12205884e+00     2.98097912e+02    -2.04400580e+00    -2.21124917e+03     4.37369880e+04   
    1.00000000e+00     5.00000000e-04    -1.12423205e+00     3.26950730e+02    -2.12751066e+00    -2.30405403e+03     4.37366541e+04   
    2.00000000e+00     1.00000000e-03    -1.13183651e+00     4.05029511e+02    -2.38240298e+00    -3.82259936e+03     4.37362710e+04   
    3.00000000e+00     1.50000000e-03    -1.13996359e+00     4.92098012e+02    -2.68948120e+00    -5.61789016e+03     4.37346621e+04   
    4.00000000e+00     2.00000000e-03    -1.14502039e+00     5.59682261e+02    -2.92172031e+00    -6.19473670e+03     4.37330328e+04   
    5.00000000e+00     2.50000000e-03    -1.14616175e+00     5.68093475e+02    -3.00201830e+00    -4.89771495e+03     4.37296853e+04   
    6.00000000e+00     3.00000000e-03    -1.14343516e+00     5.24576710e+02    -2.91944779e+00    -1.69887356e+03     4.37253906e+04   
    7.00000000e+00     3.50000000e-03    -1.13733109e+00     4.61025975e+02    -2.72840927e+00     2.61701603e+03     4.37209547e+04   
    8.00000000e+00     4.00000000e-03    -1.13043733e+00     3.82545740e+02    -2.53322454e+00     6.10257007e+03     4.37179064e+04   
    9.00000000e+00     4.50000000e-03    -1.12730355e+00     3.51177240e+02    -2.44551550e+00     7.69248060e+03     4.37165643e+04   
    1.00000000e+01     5.00000000e-03    -1.12981626e+00     3.66373231e+02    -2.51481617e+00     6.54739200e+03     4.37168849e+04   
